Loop/Views/SimpleBolusView.swift

```diff
@@ -48,8 +48,7 @@ struct SimpleBolusView: View, HorizontalSizeClassOverride {
                 // Unfortunately, after entry, the field scoots back down and remains hidden.  So this is not a great solution.
                 // TODO: Fix this in Xcode 12 when we're building for iOS 14.
                 .padding(.top, self.shouldAutoScroll(basedOn: geometry) ? -200 : 0)
-                .listStyle(GroupedListStyle()) // In iOS 14, this should be InsetGroupedListStyle()
-                .environment(\.horizontalSizeClass, .regular)
+                .insetGroupedListStyle()
```

Comment:
[nit] specific size class being used here. Is this accounted for?

Reply:
I think this code copied other code that predated `HorizontalSizeClass`